engine-tests/src/test/java/org/terasology/engine/module/ModuleManagerTest.java

```diff
@@ -21,18 +22,31 @@
             "testdummy-1.0.0-SNAPSHOT.jar",
             "messy-name-directory & $t#f/testmessy-1.0.0-SNAPSHOT.jar"
     })
-    void testLoadModuleFromURL(String jarLocation) throws IOException, URISyntaxException {
+    void testLoadModuleFromJarURL(String jarLocation) throws IOException, URISyntaxException {
         ModuleManagerImpl mm = new ThisModuleManager("");
         ModuleLoader loader = new ModuleLoader();
         loader.setModuleInfoPath(MODULE_INFO_FILENAME);
 
-        URL url = getClass().getResource("/org/terasology/engine/module/" + jarLocation);
+        URL url = getClass().getResource(jarLocation);
         assumeTrue(url != null, "test resource not found:" + jarLocation);
         URL jarUrl = new URL("jar", null, url.toString() + "!/" + MODULE_INFO_FILENAME);
 
         assertNotNull(mm.load(loader, jarUrl));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = "Spacey Parent/SourceModuleFixture/")
+    void testLoadModuleFromFileURL(String sourceLocation) throws IOException, URISyntaxException {
+        ModuleManagerImpl mm = new ThisModuleManager("");
+        ModuleLoader loader = new ModuleLoader();
+        loader.setModuleInfoPath(MODULE_INFO_FILENAME);
+
+        URL url = getClass().getResource(Paths.get(sourceLocation).resolve(MODULE_INFO_FILENAME).toString());
+        assumeTrue(url != null, "test resource not found:" + sourceLocation);
```

Comment:
`assumeTrue` does not do what you thought! it skips the test when the assumption fails!